### PR TITLE
Mypr

### PR DESCRIPTION
**Pull request by saengowp**

### CODE REVIEW

**Review by saenyakorn — APPROVED** [review]
LGTM

**Review by suphon-t — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
-# Inrto to Git
-Tutorial repository for "Intro to Git" Sharing Session
+# Intro to Git
```
Comment (saenyakorn) [inline README.md:1]:
```suggestion
# Introduction to Git
```

**Review by saenyakorn — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bombnp) [inline README.md:1]: too short bro

File: README.md [hunk 2]
```diff
@@ -1,5 +1,4 @@
...
 
 ## Table of Contents
```
Comment (saengowp) [inline README.md:3]: Pls remove this

**Review by bombnp — CHANGES_REQUESTED** [review]
your code sucks bro

**Comment by saengowp** [comment]
Hello

**Pull request closed without merging**